web-services/query-websocket/src/main/java/datawave/webservice/websocket/QueryWebsocket.java

```diff
@@ -70,36 +65,36 @@ public void closeConnection(Session session) throws IOException {
         cancelActiveQuery(session);
     }
     
-    @OnMessage
-    public void handleMessage(final Session session, QueryMessage message) {
-        switch (message.getType()) {
-            case CREATE: {
-                if (session.getUserProperties().get(ACTIVE_QUERY_FUTURE) != null) {
-                    session.getAsyncRemote().sendObject(
-                                    new QueryResponseMessage(ResponseType.CREATION_FAILURE, "Query already active. Only one query per websocket is allowed."));
-                } else {
-                    CreateQueryMessage cqm = (CreateQueryMessage) message;
-                    String logicName = (String) session.getUserProperties().get(LOGIC_NAME);
-                    QueryObserver observer = new QueryObserver(log, session);
-                    
-                    Long startTime = System.nanoTime();
-                    Long loginTime = null;
-                    try {
-                        loginTime = Long.valueOf((String) session.getUserProperties().get(REQUEST_LOGIN_TIME_HEADER));
-                    } catch (Exception e) {
-                        // Ignore -- login time won't be available
-                    }
-                    
-                    Future<?> activeQuery = queryExecutorBean.executeAsync(logicName, cqm.getParameters(), startTime, loginTime, observer);
-                    session.getUserProperties().put(ACTIVE_QUERY_FUTURE, activeQuery);
-                }
-            }
-                break;
-            case CANCEL: {
-                cancelActiveQuery(session);
-            }
-                break;
-        }
+//    @OnMessage
+    public void handleMessage(final Session session) { // , QueryMessage message) {
+    // switch (message.getType()) {
+    // case CREATE: {
+    // if (session.getUserProperties().get(ACTIVE_QUERY_FUTURE) != null) {
+    // session.getAsyncRemote().sendObject(
+    // new QueryResponseMessage(ResponseType.CREATION_FAILURE, "Query already active. Only one query per websocket is allowed."));
+    // } else {
+    // CreateQueryMessage cqm = (CreateQueryMessage) message;
+    // String logicName = (String) session.getUserProperties().get(LOGIC_NAME);
+    // QueryObserver observer = new QueryObserver(log, session);
+    //
+    // Long startTime = System.nanoTime();
+    // Long loginTime = null;
+    // try {
+    // loginTime = Long.valueOf((String) session.getUserProperties().get(REQUEST_LOGIN_TIME_HEADER));
+    // } catch (Exception e) {
+    // // Ignore -- login time won't be available
+    // }
+    //
+    // Future<?> activeQuery = queryExecutorBean.executeAsync(logicName, cqm.getParameters(), startTime, loginTime, observer);
+    // session.getUserProperties().put(ACTIVE_QUERY_FUTURE, activeQuery);
+    // }
+    // }
+    // break;
+    // case CANCEL: {
+    // cancelActiveQuery(session);
+    // }
+    // break;
+    // }
```

Comment:
Is there a good argument for leaving this commented out code in?

Reply:
I put the code back since the module is not processed by the pom and therefore not deployed. I had it commented out before I modified the pom as a means of achieving the same thing, which was to not have to update/configure code to run in the upgraded wildfly that isn't used in production.